src/uu/rev/src/rev.rs

```diff
@@ -68,11 +79,18 @@ pub fn uu_app() -> Command {
         .override_usage(format_usage(USAGE))
         .infer_long_args(true)
         .arg(
-            Arg::new("file")
+            Arg::new(options::FILE)
                 .value_name("FILE")
                 .help("Paths of files to reverse")
                 .index(1)
                 .action(ArgAction::Set)
                 .num_args(1..),
         )
+        .arg(
+            Arg::new(options::ZERO)
+                .short('0')
+                .long("zero")
+                .help("Zero termination. Use the byte '\\0' as line separator.")
+                .action(ArgAction::SetFalse),
```

Comment:
Why do you use `SetFalse` instead of `SetTrue`?

Reply:
fixed